internal/winapi/process.go

```diff
@@ -0,0 +1,100 @@
+package winapi
+
+import "syscall"
+
+const PROC_THREAD_ATTRIBUTE_PARENT_PROCESS = 0x00020000
+
+const PROCESS_ALL_ACCESS uint32 = 0x1FFFFF
+
+// typedef struct _PROC_THREAD_ATTRIBUTE {
+//     ULONG_PTR Attribute;
+//     SIZE_T Size;
+//     ULONG_PTR Value;
+// } PROC_THREAD_ATTRIBUTE, *PPROC_THREAD_ATTRIBUTE;
```

Comment:
same with these definitions?

Reply:
Yes I feel we should do these for all of our bindings. Makes it easier when going to make future additions to winapi to see what go types map to what windows types. Avoids unnecessary web searches